src/main/java/uk/ac/cam/cl/dtg/segue/api/EmailFacade.java

```diff
@@ -496,4 +498,109 @@ public final Response sendEmailsToUserIds(@Context final HttpServletRequest requ
 
         return Response.ok().build();
     }
+
+    /**
+     * sendemailwithuserids allows sending an email to a given list of userids.
+     *
+     * This method will return serialised html that displays an email object
+     *
+     * @param request
+     *            - so that we can allow only logged in users to view their own data.
+     * @param emailTypeString
+     *            - the type of e-mail that is being sent.
+     * @param emailTemplates
+     *            - map which must contain the plaintextTemplate and htmlTemplate
+     * @return Response object containing the serialized content object. (with no levels of recursion into the content)
+     */
+    @POST
+    @Path("/email/sendcontentemailwithuserids/{emailtype}")
```

Comment:
Could this be `sendprovidedemailwithuserids`? I assumed `content` meant from the content repo and got confused!